### ISSUE TITLE
Almost no benefit of func create

### ISSUE THREAD

**Issue by xxm0703** [semerdzhiev/sdp-2021-22#5]
Isn't it supposed to use **create** when constructing a new obj for every test case, rather than `Hierarchy h(...);`?

https://github.com/semerdzhiev/sdp-2021-22/blob/654003a151852ba05120455831442843ae24c771/tests/2/tests.cpp#L34

**Comment by poseidon4o** [comment]
True, it is used only twice. You can make PR to remove it if you want.
